src/api/database/bulk-api.rst

```diff
@@ -167,7 +158,7 @@
     database. The information is returned as a JSON structure containing meta
     information about the return structure, including a list of all design
     documents and basic contents, consisting the ID, revision and key. The key
-    is the from the design document's ``_id``.
+    is from the design document's ``_id``.
```

Comment:
Would this be more clear as

```suggestion
    is the design document's ``_id``.
```

? The current wording sounds as if the key may be some how derived from, but not equal to, the `_id`.

Reply:
Good call! I was so caught up on the double `the` that I didn't realize this might still be misleading. Just applied your suggestion.